[2023-10-07T03:09:47.892+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_sql_dag.check_connect_to_db manual__2023-10-07T03:09:45.603070+00:00 [queued]>
[2023-10-07T03:09:47.905+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_sql_dag.check_connect_to_db manual__2023-10-07T03:09:45.603070+00:00 [queued]>
[2023-10-07T03:09:47.906+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 2
[2023-10-07T03:09:47.926+0000] {taskinstance.py:1380} INFO - Executing <Task(PythonOperator): check_connect_to_db> on 2023-10-07 03:09:45.603070+00:00
[2023-10-07T03:09:47.935+0000] {standard_task_runner.py:57} INFO - Started process 683 to run task
[2023-10-07T03:09:47.941+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'etl_sql_dag', 'check_connect_to_db', 'manual__2023-10-07T03:09:45.603070+00:00', '--job-id', '950', '--raw', '--subdir', 'DAGS_FOLDER/etl_sql_dag.py', '--cfg-path', '/tmp/tmptsuz8vk_']
[2023-10-07T03:09:47.946+0000] {standard_task_runner.py:85} INFO - Job 950: Subtask check_connect_to_db
[2023-10-07T03:09:48.011+0000] {task_command.py:415} INFO - Running <TaskInstance: etl_sql_dag.check_connect_to_db manual__2023-10-07T03:09:45.603070+00:00 [running]> on host e7b1a9bbd1dc
[2023-10-07T03:09:48.114+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_sql_dag' AIRFLOW_CTX_TASK_ID='check_connect_to_db' AIRFLOW_CTX_EXECUTION_DATE='2023-10-07T03:09:45.603070+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-10-07T03:09:45.603070+00:00'
[2023-10-07T03:09:48.166+0000] {logging_mixin.py:151} INFO - 19.3.0.0.0
[2023-10-07T03:09:48.173+0000] {logging_mixin.py:151} INFO - connected to database successful
[2023-10-07T03:09:48.191+0000] {python.py:194} INFO - Done. Returned value was: True
[2023-10-07T03:09:48.215+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=etl_sql_dag, task_id=check_connect_to_db, execution_date=20231007T030945, start_date=20231007T030947, end_date=20231007T030948
[2023-10-07T03:09:48.274+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-10-07T03:09:48.311+0000] {taskinstance.py:2776} INFO - 1 downstream tasks scheduled from follow-on schedule check
